pom.xml

```diff
@@ -213,7 +213,7 @@
       8.43
     </maven.sevntu-checkstyle-check.checkstyle.version>
     <maven.versions.plugin.version>2.8.1</maven.versions.plugin.version>
-    <java.version>1.8</java.version>
+    <java.version>11</java.version>
```

Comment:
This is commit is confusing as this line does more than update the CI. It is enforcing all checkstyle to be Java 11.
Either the commit should be renamed or this PR should be split into 2.
I think it makes more sense to have 1 PR for upgrading the CI and another for enforcing all checkstyle to be Java 11.

Reply:
I think we targeting full migration to jdk11.
And this is draft PR, so some items are still not done

Reply:
We can split in few PRs but I do not see this PR to be big and require split.

I can migrate TC to jdk11, as not that much people run inspection on local, it can be upfront for few months

Reply:
Yes, this is just a probe.
I'll split it to multiple PRs. The last one should contain only one change in the  pom.xml file.

Reply:
We do not have to split, I think one PR will be good.